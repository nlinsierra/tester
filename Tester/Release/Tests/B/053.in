JH 9S 3C 5D QC 3S 8C 6S 5S 4S 6H 2C JD 2H 9C 7C KS 4C 4D 7H JC 3D TS 7S
