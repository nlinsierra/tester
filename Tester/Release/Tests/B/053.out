28
5C 6C TC KC AC 2D 6D 7D 8D 9D TD QD KD AD 3H 4H 5H 8H 9H TH QH KH AH 2S 8S JS QS AS
